CHANGELOG.md

```diff
@@ -9,9 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 * Added Zenodo badge to README
+* Pagination of results for search methods of `NanopubClient`
 
 ### Changed
 * `nanopub-java` dependency is installed upon installation instead of upon runtime.
+* search methods of `NanopubClient` return iterator instead of list
```

Comment:
In theory this is a backwards incompatible API change, so we should release to v2.0.0. I also don't want to make this backwards compatible. What do you think @raar1?

Reply:
I agree, let's have it as a backwars incompatible change this time round. I think we only have a couple of users so far so it should be ok to make this change and communicate it to them. In future of course (especially after the blog post) we will want to avoid such changes, I guess. I am tempted simply to increment by the usual 0.1, unless you strongly object?

Reply:
Yes, the new version shall be 1.2.0